docs/src/contrib.md

```diff
@@ -2,23 +2,31 @@
 
 If you are interested in contributing to this package,
 please consider going through this guide to help make your
-developing workflow smooth.
+development workflow as smooth as possible.
 
-Contributing to the documentation is always a good start to get yourself familiar
+Contributing to documentation and unit tests are always great ways to get yourself familiar
 with the community and workflows.
```

Comment:
"unit tests" is a little unfamiliar to me. @Roger-luo , could you take a look of this point?